uint64 timestamp # time since system start (microseconds)

bool esc_flashing_in_progress

uint8 version_major

uint8 version_minor
